### ISSUE TITLE
Looking for the dataset csuite_fork_collider_nonlin_gauss_latent_confounder

### ISSUE THREAD

**Issue by fred887** [microsoft/causica#97]
Hello,

I would like to reproduce the results of the **D-DECI** method (from v0.0.0 version) on the synthetic data in the corresponding paper.\
But I cannot find the `csuite_fork_collider_nonlin_gauss_latent_confounder` dataset.\
Neither here in https://github.com/microsoft/causica nor in https://github.com/microsoft/csuite.

Could you please indicate me where I can get this dataset or maybe provide me some source code to generate it by myself?

Thank you very much for your help,

Bests,

Frederic

**Comment by KaushikSathvara** [comment]
@fred887 Did you get any chance to find dataset? 
 `csuite_fork_collider_nonlin_gauss_latent_confounder`

**Comment by KaushikSathvara** [comment]
Yeah, I tried that as well. But seems it's throwing me some runtime error like below.

`RuntimeError: mat1 and mat2 must have the same dtype`

**Comment by LaurantChao** [comment]
Hi Frederic, 

Thanks for reaching out. 

I have now updated the csuite data to include `csuite_fork_collider_nonlin_gauss_latent_confounder`, which should be accessible (please refer to [microsoft/csuite: CSuite: A Suite of Benchmark Datasets for Causality (github.com)](https://github.com/microsoft/csuite).

Best,
Chao

**Comment by fred887** [comment]
Hi Chao,
Thank you very much!
Best,
Frederic
